src/main/java/com/akukhtin/springbootapplication/controller/UserController.java

```diff
@@ -0,0 +1,60 @@
+package com.akukhtin.springbootapplication.controller;
+
+import com.akukhtin.springbootapplication.dto.UserDto;
+import com.akukhtin.springbootapplication.dto.UserDtoUtil;
+import com.akukhtin.springbootapplication.exception.UserIdMismatchException;
+import com.akukhtin.springbootapplication.model.User;
+import com.akukhtin.springbootapplication.repository.UserRepository;
+import java.util.Optional;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@Controller
+@RequestMapping("/api/users")
+public class UserController {
+    @Autowired
+    private UserRepository userRepository;
+
+    @GetMapping
+    public Iterable findAll() {
+        return userRepository.findAll();
+    }
+
+    @GetMapping("/{id}")
+    public Optional<User> findOne(@PathVariable Long id) {
```

Comment:
```suggestion
    public User findOne(@PathVariable Long id) {
```
@AlexKukhtinDev we should not return the optional in the controller layer. Please read more about this why and explain it to me)